src/_includes/docs/swift-package-manager/migrate-macos-project-manually.md

```diff
@@ -0,0 +1,84 @@
+Once you [turn on Swift Package Manager][] support, the Flutter CLI tries to
+migrate your project to add Swift Package Manager integration.
+This makes your project download the Swift packages that your Flutter
+plugins depend on.
+
+However, the Flutter CLI might be unable to migrate your project if you've
```

Comment:
same: "even if" , here, right?

Reply:
I dropped the "significantly" qualifier and switched to wording similar to what you suggested here: https://github.com/flutter/website/pull/10827#discussion_r1680007985